src/Compilers/VisualBasic/Test/Emit/PDB/PDBTests.vb

```diff
@@ -66,6 +66,38 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.PDB
 
         End Sub
 
+        <Fact>
```

Comment:
Is this test related to the current change?

Reply:
Yes, same as C#.